README.md

```diff
@@ -37,7 +37,7 @@ many users will want iris and/or xarray as well:
 
 
 ```
-conda install -c conda-forge -c ioam holoviews>=1.6.2 geoviews
```

Comment:
Well, you get the latest version unless that conflicts with some other package's requirement, which I guess we're counting on not happening in this case.